logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -216,3 +229,88 @@ func stringToLogLevel(level string) logging.LogLevel {
 
 	return logging.InfoLevel
 }
+
+const (
+	HookSysLog   = "syslog"
+	HookLogStash = "logstash"
+	HookFluent   = "fluent"
+)
+
+type levelMapping struct {
+	syslogLevel map[logging.LogLevel]syslog.Priority
+	fluentLevel map[logging.LogLevel][]logrus.Level
+}
+
+type HookConfig struct {
+	Protocol string
+	Address  string
+	Port     int
+	Level    string
+}
+
+var llm = levelMapping{}
+
+func createLevelMappings() {
+	llm.syslogLevel = map[logging.LogLevel]syslog.Priority{
+		logging.PanicLevel: syslog.LOG_CRIT,
```

Comment:
This kind of conversion already happens inside logrus's syslog hook package: https://github.com/sirupsen/logrus/blob/master/hooks/syslog/syslog.go#L35-L49

Reply:
I need these conversions when creating hooks.

Reply:
Have you tested how it currently behaves? I believe that if I set hook's level to `panic` now, it will log with `syslog.LOG_CRIT` to syslog.. but it will log ALL levels! 
See my latest review.

Reply:
Please, if you have trouble grasping how logrus behaves. Create small separate project to test out all combinations until you are certain with its behavior. Only then continue with implementation here.